extensions/mssql/src/main.ts

```diff
@@ -340,17 +340,21 @@ async function handleOpenNotebookTask(profile: azdata.IConnectionProfile): Promi
 }
 
 async function handleOpenClusterStatusNotebookTask(profile: azdata.IConnectionProfile, appContext: AppContext): Promise<void> {
-	const notebookRelativePath = 'notebooks/tsg/cluster-status.ipynb';
-	const notebookFullPath = path.join(appContext.extensionContext.extensionPath, notebookRelativePath);
+	const notebookRelativePath: string = 'notebooks/tsg/cluster-status.ipynb';
+	const notebookFullPath: string = path.join(appContext.extensionContext.extensionPath, notebookRelativePath);
 	if (!Utils.fileExists(notebookFullPath)) {
 		vscode.window.showErrorMessage(localize("fileNotFound", "Unable to find the file specified"));
```

Comment:
I know this is not changed in this PR, but I would recommend more specific message here for example: Unable to find the notebook: notebook path. makes it easier for both user and us when troubleshooting.